enhancements/hypershift/performanceprofilecontroller-node-tuning.md

```diff
@@ -0,0 +1,328 @@
+---
+title: performanceprofilecontroller-node-tuning
+
+authors:
+  - "@jlojosnegros"
+
+reviewers:
+  - "@dagrayvid"
+  - "@jmencak"
+  - "@csrwng"
+  - "@enxebre"
+  - "@sjenning"
+  - "@alvaroaleman"
+  - "@derekwaynecarr"
+  - "@imain"
+
+approvers:
+  - "@csrwng"
+  - "@enxebre"
+  - "@sjenning"
+  - "@alvaroaleman"
+  - "@derekwaynecarr"
+  - "@imain"
+
+api-approvers:
+  - None
+
+tracking-link:
+  - https://issues.redhat.com/browse/PSAP-742
+  
+see-also:
+  - "/enhancements/hypershift/node-tuning.md"
+
+creation-date: 2022-09-19
+last-updated: 2022-09-19
+---
+
+# Performance Profile Controller Adaptation to Hypershift
+
+## Summary
+
+This enhancement describes how the [Performance Profile Controller](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_controller.md), previously known as Performance add-on operator,
+will run in Hypershift's managed control plane.
+
+This is part of Phase-3 for Node Tunning Operator enablement and so it depends on the two previous phases.
+
+## Motivation
+
+See the HyperShift [project goals](https://hypershift-docs.netlify.app/reference/goals-and-design-invariants/).
+These goals should not come at the cost of giving up any existing OpenShift features.
+
+In standalone OpenShift Performance Profile Controller, as part of Node Tuning Operator, helps in optimizes OpenShift clusters for applications sensitive to cpu and network latency
+by letting the user to create a more comprehensive and unify API from where the controller extract the appropiate parameters,
```

Comment:
s/unify/unified/
s/where the controller extract/where the controller extracts/

Reply:
Done, thanks for your review.